chart/values.yaml

```diff
@@ -136,3 +136,19 @@ customLogos:
   # storageClass: "-"
   accessMode: ReadWriteOnce
   size: 1Gi
+
+# Rancher post-delete hook
+postDelete:
+  enabled: true
+  name: rancher-post-delete
+  image:
+    repository: %POST_DELETE_IMAGE_NAME%
+    tag: %POST_DELETE_IMAGE_TAG%
+  namespaceList:
+    - fleet-system
+    - cattle-system
+    - rancher-operator-system
```

Comment:
any reason why we expose this instead of hard-coding the namespaceList in the chart deployment? Is there a use case where a suer can install stuff onto non-default namespaces that we want to support?

Reply:
Yes I think Raul requested that. Can also be used to clean up other resources, or not clean up fleet.